deploy-all.js

```diff
@@ -11,12 +11,12 @@ function getPlaceholderFromPath(libPath) {
   return modifiedPath.slice(0, 36);
 }
 
-async function main() {
+async function deployKovan() {
   try {
     const networkName = 'kovan';
     const web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'));
     const accounts = await web3.eth.getAccounts();
-    const opts = { from: accounts[0], gas: 6900000 };
+    const opts = { from: accounts[0], gas: 6900000, gasPrice: 100000000000 };
```

Comment:
for those literals would do config file - eg something like [here](https://github.com/melonproject/microservices/blob/master/src/conf.js)

w configs dependent on `networkArgument`